src/Components/Reports/Common/ExecutiveFirstPage.js

```diff
@@ -49,33 +46,37 @@ export const ExecutiveFirstPage = ({ data }) => {
 
     const { stateChartData, stateTableData, conditionsChartData,  conditionsTableData, condtionsInfo } = formattedReportData;
 
-    ioOccuranceTableData.push(...condtionsInfo.io.occurances);
-    ramOccuranceTableData.push(...condtionsInfo.memory.occurances);
-    cpuOccuranceTableData.push(...condtionsInfo.cpu.occurances);
+    ioOccuranceTableData.push(...condtionsInfo.io.occurrences);
```

Comment:
Same here (line 49 to 50)
```suggestion
    ioOccurenceTableData.push(...condtionsInfo.io.occurrences);
```

Reply:
fixed it in latest commit. Thanks @r14chandra!